include/xlnt/utils/date.hpp

```diff
@@ -49,21 +50,54 @@ struct XLNT_API date
     static date from_number(int days_since_base_year, calendar base_date);
 
     /// <summary>
-    /// Constructs a data from a given year, month, and day.
+    /// Constructs a date from a given year, month, and day.
     /// </summary>
     date(int year_, int month_, int day_);
 
     /// <summary>
-    /// Return the number of days between this date and base_date.
+    /// Constructs an empty date (a call to is_null() will return true).
+    /// </summary>
+    date() = default;
+
+    /// <summary>
+    /// Return the number of days between this date and base_date. The date could be in an empty/invalid state, so you may want to call is_null() first!
```

Comment:
As we are changing this line, it may be a good time to fix the grammar => return**s**?

Reply:
I fixed the grammar there and in a few other places related to date/time functions :+1: